server/templates/obs.html

```diff
@@ -4,26 +4,35 @@
 {% block content %}
     <h3>Observation  {{ obs.obs_id }}</h3>
 
-    <p>
-        Thumbnail:<br/>
+    <p style="float: right">
         <img src="/data/thumbs/{{ obs.thumbnail }}" />
     </p>
+
+    <h4>Transmission parameters</h4>
     <p>
         <dl>
-            <dt>AOS:</dt><dd>{{ obs.aos }}</dd>
-            <dt>TCA:</dt><dd>{{ obs.tca }}</dd>
-            <dt>LOS:</dt><dd>{{ obs.los }}</dd>
+            <dt class="badge">
```

Comment:
I understand this class, because it looks nice? ```badge``` class is for notification counters - elements which are shown occasionally .

In Bootstrap different CSS classes has many relationships, often not clear. If you use the Boostrap components in other context as dedicated then it may cause hard to resolve UI bugs (for example some elements will be overlapped shifted by single pixels).

Reply:
Yup, looked nice and I wanted to make it somewhat similar to what satnogs has on their page. Would it be better if I defined my own class with exactly the same params?

Reply:
Yes, because you copy only properties, but not chain of inheritance and related behaviors.
```<dt>``` will be look as "badge", but it will be not a "badge".

Reply:
Makes sense. Renamed badge to param.